module-apps/application-settings/windows/display-keypad/WallpaperWindow.cpp

```diff
@@ -39,7 +39,7 @@ namespace gui
                 },
                 [=](gui::Item &item) {
                     if (item.focus) {
-                        this->setNavBarText(utils::translate(style::strings::common::Switch), nav_bar::Side::Center);
+                        this->setNavBarText(utils::translate(style::strings::common::select), nav_bar::Side::Center);
```

Comment:
strange that there are fields with mixed upper and lower cases...

Reply:
Yes, it is a bit strange. It is only the case for Switch. I suppose it is to avoid syntax keyword.

Reply:
alright, that's fine then ;)